test/integration/prerender/test/index.test.js

```diff
@@ -405,6 +405,38 @@ const runTests = (dev = false, looseMode = false) => {
     }
   })
 
+  it('should support nested lazy catchall route', async () => {
+    // Dev doesn't support fallback yet
+    if (dev) {
+      const html = await renderViaHTTP(
+        appPort,
+        '/catchall/notreturnedinpaths/nested'
+      )
+      const $ = cheerio.load(html)
+      expect($('#catchall').text()).toMatch(/Hi.*?notreturnedinpaths nested/)
+    }
+    // Production will render fallback for a "lazy" route
+    else {
+      const html = await renderViaHTTP(
+        appPort,
+        '/catchall/notreturnedinpaths/nested'
+      )
+      const $ = cheerio.load(html)
+      expect($('#catchall').text()).toBe('fallback')
+
+      // hydration
+      const browser = await webdriver(appPort, '/catchall/delayby3s/nested')
+
+      const text1 = await browser.elementByCss('#catchall').text()
+      expect(text1).toBe('fallback')
+
+      await new Promise(resolve => setTimeout(resolve, 4000))
```

Comment:
```suggestion
      await waitFor(4000)
```
or should we use the `check` util we have here instead of trying to guess the time it should appear by?

Reply:
This was copied from the existing test. I can update both.

Reply:
Updated.